nltk/text.py

```diff
@@ -173,36 +171,47 @@ def __repr__(self):
         return '<ConcordanceIndex for %d tokens (%d types)>' % (
             len(self._tokens), len(self._offsets))
 
-    def print_concordance(self, word, width=75, lines=25):
-        """
-        Print a concordance for ``word`` with the specified context window.
+    def __concordance__(self, word, width=80):
+        """Generate a concordance list of tuples(left, token, right)
```

Comment:
Dunder functions are little pricky. Try not to create new ones that are non-canonical call them something else, e.g. `generate_concordance()` or anything without a dunder.

See @amontalenti guide on http://www.pixelmonkey.org/2013/04/11/python-double-under-double-wonder